src/main/java/com/iota/iri/Iota.java

```diff
@@ -50,61 +44,64 @@
     public final int tcpPort;
     public final int maxTipSearchDepth;
 
-    public Iota(Configuration configuration) {
+    public Iota(Configuration configuration) throws IOException {
         this.configuration = configuration;
         testnet = configuration.booling(Configuration.DefaultConfSettings.TESTNET);
         maxPeers = configuration.integer(Configuration.DefaultConfSettings.MAX_PEERS);
         udpPort = configuration.integer(Configuration.DefaultConfSettings.UDP_RECEIVER_PORT);
         tcpPort = configuration.integer(Configuration.DefaultConfSettings.TCP_RECEIVER_PORT);
 
+        // MAKE INITIAL SNAPSHOT
         String snapshotFile = configuration.string(Configuration.DefaultConfSettings.SNAPSHOT_FILE);
         String snapshotSigFile = configuration.string(Configuration.DefaultConfSettings.SNAPSHOT_SIGNATURE_FILE);
-        Snapshot initialSnapshot = Snapshot.init(snapshotFile, snapshotSigFile, testnet).clone();
+        Snapshot initialSnapshot = Snapshot.init(snapshotFile, snapshotSigFile, testnet)
```

Comment:
The real change here is that since the IOException is propagated, there will no longer be a hidden RuntimeException (previous) after the `false` flag on IOException in SignedFiles and Snaphot. Effectively, the exception would have been propagated anyways, albeit invisible to callers.